internal/handlers/web/index.html

```diff
@@ -47,6 +47,10 @@ <h2 class="font-bold text-lg mb-4">Generated Lines</h2>
 			<!-- Dynamically generated number elements will be styled similarly to the selected numbers -->
 		</div>
 	</div>
+	<hr>
+	<footer class="text-center text-sm p-4">
```

Comment:
Consider adding some styling to the footer to make it visually distinct from the rest of the page. This could be a different background color or a horizontal line above the footer. [medium]